### PR TITLE
shared memory cards finished (#5)

### LINKED ISSUE #5: Cards: Shared Memory
We will need to write some cards for the "Shared Memory" slides. Anyone up for the task can assign this issue to him/herself.

### PR DESCRIPTION
**Pull request by Benjaminimal**

### CODE REVIEW

File: shmem_basic.txt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  <li>Consists of a fixed number of processor-cores <i>p</i></li>
+  <li>The cores are connected to a large shared memory</li>
+  <li>Every core can read/write every location in memory</li>
+  <li>Memory access time is not uniform (NUMA)</li>
+  <li>Processors are not synchronized</li>
+</ul>";shared_memory models
+Symmetric Multiprocessing (SMP);All cores are equal, OS decides when and where to run a process (thread);shared_memory definition
+What are the benefits and drawbacks of Symmetric Multiprocessing (SMP)?; The OS can exploit the cores well but sometimes it is better to pin a process to a core (e.g. private cache utilization);shared_memory cache
+What is the main difference between real shared-memory systems and their models?; Real systems use caches;shared_memory cache
+Cache coherency problem;What happens when processors read/write the same address?;shared_memory cache
```
Comment (hurbeana) [inline shmem_basic.txt]: I think this is the wrong way around, isn't it? (the line with the Cache coherency problem)

**Review by hurbeana — COMMENTED** [review]
So I looked through everything and check if everything is fine in Anki. Except for the one possible little mistake the rest is very good! Thank you :) If you could just swap the fields in that one line I'll merge it into the project and do a new release!

If you need help on putting another change to the PR, [heres how](https://stackoverflow.com/a/9790530)

File: shmem_basic.txt (hunk 1 above)
Comment (Benjaminimal) [reply shmem_basic.txt]: I can change it but I don't see what you mean here, sorry.

Comment (hurbeana) [reply shmem_basic.txt]: I think it should be: front side=Question, back side=Answer, like you have in other cards. Imo it's the other way around here.

Comment (Benjaminimal) [reply shmem_basic.txt]: Now I see what you mean. I'll change it for the next card about the memory consistency problem as well.

Comment (hurbeana) [reply shmem_basic.txt]: Thank you, that fixed it!

**Comment by hurbeana** [comment]
Thank you for the great additions to the cardset! Closes #5

**Pull request merged**